src/utils/plotSpecs/generateFrequencySpec.js

```diff
@@ -156,7 +156,7 @@ const generateSpec = (config, plotData, xNamesToDisplay, yNamesToDisplay) => {
         orient: 'bottom',
         scale: 'x',
         zindex: 1,
-        title: config.axes.xAxisText,
+        title: config.xAxisGrouping,
```

Comment:
Why this rename? how was `xAxisGrouping` different from `xAxisText` before or were they the same?

Reply:
now for title we use the value of `xAxisGrouping`, which points to the name of the hierarchy. `axes.xAxisText` was a hardcoded value in the initial state.

Reply:
Good point @cosa65 I brought `xAxisText` back and I am setting it to the label of the selected option. What do you think?